.circleci/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
 
       - run: &install-package
           name: Install package
-          command: env/bin/pip install .
+          command: env/bin/pip install -e .
```

Comment:
Why the change to an editable install here?

Reply:
For coverage to work correctly, same as [here](https://github.com/dwavesystems/dwave-cloud-client/commit/3cb5ab65c81d146d801ac1b39c66ad31b67c643f). Otherwise, the `coverage` will report on files in `site-packages`, not source file. (Only happens with `coverage` of pep420 packages.)